tests/framework/db/CommandTest.php

```diff
@@ -809,17 +809,34 @@ public function testAutoRefreshTableSchema()
     {
         $db = $this->getConnection(false);
         $tableName = 'test';
+        $fkName = 'test_fk';
+
+        $this->assertNull($db->getSchema()->getTableSchema($tableName));
 
         $db->createCommand()->createTable($tableName, [
             'id' => 'pk',
+            'fk' => 'int',
             'name' => 'string',
         ])->execute();
         $initialSchema = $db->getSchema()->getTableSchema($tableName);
+        $this->assertNotNull($initialSchema);
 
         $db->createCommand()->addColumn($tableName, 'value', 'integer')->execute();
         $newSchema = $db->getSchema()->getTableSchema($tableName);
         $this->assertNotEquals($initialSchema, $newSchema);
 
+        $db->createCommand()->addForeignKey($fkName, $tableName, 'fk', $tableName, 'id')->execute();
```

Comment:
https://travis-ci.org/yiisoft/yii2/jobs/226569561#L325
I think some tests is better to mark skipped for some particular DBMS (like https://github.com/yiisoft/yii2/blob/master/tests/framework/db/CommandTest.php#L561)